CONTRIBUTING.md

```diff
@@ -0,0 +1,52 @@
+**Thanks for taking the time to contribute! ❤️**
+
+All types of contributions are encouraged and valued, no matter if it's a bug
+report 🐛, a feature request 💡, or a Pull Request 🚀.
+
+- **❓ I have a question:** [Open an Issue]
+- **🐛 I found a bug:** [Open an Issue]
+- **💡 I have an idea:** [Open an Issue]
+- **💻 I want to code:** See below
+
+If you like the project, but just don't have time to contribute, that's OK too!
+You can also star the project ⭐, rave about it online 💬, or add a link to us
+🔗 in your project's readme.
+
+⚠️ You must never report security 🔒 related issues, vulnerabilities or bugs
```

Comment:
I'm not sure if we need this. If it's a Microsoft-related security issue, this makes sense, but security issues may be beyond Microsoft as dev containers are more agnostic.

cc @chrmarti @alexdima for any input